package.json

```diff
@@ -43,6 +43,7 @@
     "@babel/core": "^7.0.0",
     "@babel/plugin-transform-runtime": "^7.1.0",
     "@babel/preset-env": "^7.0.0",
+    "aws-sdk": "^2.372.0",
```

Comment:
We should remove the `"s3": "^4.4.0"` dependency if we no longer use it.

Reply:
It seems it is still being used in `scripts/branch/publish.js` (vs scripts/publish.js)... That is a quite similar script (not currently gzipped) just for branches, so I'm gonna refactor them both and try to remove the dependency

Reply:
Sorry, I didn't check it. 😄 

Much better if you do so. 💪